Directory.Build.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <ItemGroup>
+    <Protobuf Condition="'$(DisableAspNetCoreDefaultClientTypeOverride)' != 'true'" Update="@(Protobuf)">
+      <ClientBaseType Condition="'%(Protobuf.GrpcServices)' == 'Client' OR '%(Protobuf.GrpcServices)' == 'Both'">LiteClientBase</ClientBaseType>
```

Comment:
why do we need this?

Reply:
Got it, this sets the clientType to "LiteClientBase" by default, unless overridden. Perhaps we should add a comment, it's not that obvious what it does.

Also, is this a pattern we're going to be recommending to users?

Reply:
No, we include a target file in Grpc.Net.Client with this logic so anyone using that package (i.e. using the managed client) will get the override. The logic is that if you are referencing the manage client package AND you don't explicitly disable the override with `DisableAspNetCoreDefaultClientTypeOverride` AND the protobuf reference is for Both or Client, we'll override it to use LiteClientBase instead of ClientBase.